src/Adapter/Cart/CartPresenter.php

```diff
@@ -230,6 +230,12 @@ public function present(Cart $cart)
             'amount' => $this->pricePresenter->convertAndFormat($this->includeTaxes() ? $total_including_tax : $total_excluding_tax)
         ];
 
+        $subtotals['products'] = [
+            'type'   => 'products',
+            'label'  => $this->translator->trans('Products', [], 'Cart'),
+            'amount' => $total['amount'] - $subtotals['shipping']['amount']
```

Comment:
'amount' is supposed to be a string, with the subtotal expressed in the customer monetary units and with the currency sign already set, please see for instance how the 'shipping' subtotal is computed.